src/http-exports.md

```diff
@@ -1,4 +1,21 @@
 ---
 ---
 
-# Exports HTTP Interface
\ No newline at end of file
+# Exports HTTP Interface
+
+This holds queries which are then used to export data form learning locker.
+
+Can be accessed through the [connection API](/http-connection) and [restify api](/http-models).
+
+### Schema
+
+Name | Description
+--- | ---
+_id | The unique id of document.
+organisation | The organisation id this export template belongs to.
+name | Title of this export template.
+owner | The id of the [user](/http-users#schema) who created this export template.
+projection | an array of stringified json mongo projection queries. [See mongo docs](https://docs.mongodb.com/manual/reference/operator/aggregation/project/)
```

Comment:
Capital "A" in "an" because it's the start of a sentence and consistent with previous columns in this table.

Reply:
Done.